pex/interpreter.py

```diff
@@ -3,7 +3,7 @@
 
 """pex support for interacting with interpreters."""
 
-from __future__ import absolute_import
+from __future__ import absolute_import, print_function
```

Comment:
Maybe I'm missing something, but I don't see any print() in this file?

Reply:
I'm a print-based debugger; so you'll often see these leftover when I needed to debug against Python 2.7 and forgot to clean up.

Fixed.